shop/shop/src/main/java/ro/msg/learning/shop/repository/ProductCategoryRepository.java

```diff
@@ -0,0 +1,10 @@
+package ro.msg.learning.shop.repository;
+import org.springframework.data.jpa.repository.JpaRepository;
+import ro.msg.learning.shop.model.ProductCategory;
+
+import java.util.Optional;
+
+public interface ProductCategoryRepository  extends JpaRepository<ProductCategory, Integer> {
+    Optional<ProductCategory> findById(Integer id);
```

Comment:
The jpa repository should already have this find by id function, you don't have to redeclare it